integration_tests/src/main/python/repart_test.py

```diff
@@ -15,15 +15,35 @@
 import pytest
 
 from asserts import assert_gpu_and_cpu_are_equal_collect
+from spark_session import is_before_spark_311
 from data_gen import *
 from marks import ignore_order
 
-@pytest.mark.parametrize('data_gen', all_gen, ids=idfn)
+nested_scalar_mark=pytest.mark.xfail(reason="https://github.com/NVIDIA/spark-rapids/issues/1459")
```

Comment:
It wasn't to save the line length. I did it primarily to avoid repeating the pytest.mark every time when it was exactly the same. If you want I can change this but I think its better this way.